scripts/build-bmo-push-data.pl

```diff
@@ -173,7 +173,18 @@ sub fetch_bug {
   die 'missing id' unless $bug_id;
 
   my $response = _get('bug/' . $bug_id, {include_fields => BUG_FIELDS,});
-  return $response->{bugs}->[0];
+
+  if (exists $response->{bugs}) {
+    return $response->{bugs}->[0];
+  }
+  else {
+    return {
+      bug_id     => $bug_id,
+      status     => 'RESOLVED',
+      resolution => 'FIXED',
+      summary    => '*** Place holder for summary ***'
+    };
+  }
```

Comment:
Is there a way that we can specifically provide this information if the bug is indeed private, rather than for every case where we get a `4xx` error when accessing the bug?

Reply:
I can look for the specific 4xx code that happens when a bug is private instead. Will update.